ci/kokoro/docker/common.cfg

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 build_file: "google-cloud-cpp/ci/kokoro/docker/build.sh"
-timeout_mins: 120
+timeout_mins: 60
```

Comment:
This does not have the desired effect. This reduces the timeout for the full build to 60 minutes.